x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/action/DeleteTrainedModelAssignmentAction.java

```diff
@@ -18,11 +18,11 @@
 import java.io.IOException;
 import java.util.Objects;
 
-public class DeleteTrainedModelAllocationAction extends ActionType<AcknowledgedResponse> {
-    public static final DeleteTrainedModelAllocationAction INSTANCE = new DeleteTrainedModelAllocationAction();
+public class DeleteTrainedModelAssignmentAction extends ActionType<AcknowledgedResponse> {
+    public static final DeleteTrainedModelAssignmentAction INSTANCE = new DeleteTrainedModelAssignmentAction();
     public static final String NAME = "cluster:internal/xpack/ml/model_allocation/delete";
```

Comment:
Renaming the actions broke WAY too much stuff. So, I didn't rename these internal actions.